### DISCUSSION TITLE
Message selection when exporting chats

### DISCUSSION THREAD

**Discussion by gfd2020** [sepinf-inc/IPED#2066] [General]
I think PA can export chats from selected messages only (unless I'm wrong).
This is useful for large chats where the desired content is in just a small chunk. However, this can generate texts with the wrong context. Wouldn't there be a way to have the option of exporting only the necessary parts of the conversation and minimally preserving the context?

**Comment by lfcnassif** [comment]
It was already asked before once or twice. But I think it wouldn't be easy to implement, could have a performance impact, and given the misinterpretation risk, it wasn't a priority in our ~250 tasks queue...
